spec/liquid_tags/replit_tag_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe ReplitTag, type: :liquid_tag do
   describe "#id" do
     let(:replit_id) { "@WigWog/PositiveFineOpensource" }
+    let(:invalid_replit_id) { "@Wig-Wog/PositiveFineOpensource" }
```

Comment:
It took me a couple minutes to figure out what was different here, like those "spot the differences between these images" games in the children's magazines I used to read when I went to the dentist.

This change should make it a bit more obvious what's going on:

```suggestion
    let(:replit_id) { "@WigWog/PositiveFineOpensource" }
    let(:invalid_replit_id) { "@Cant-Have-Dashes/PositiveFineOpensource" }
```